config/environments/test.rb

```diff
@@ -31,24 +34,19 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
-  # Don't care if the mailer can't send
-  config.action_mailer.raise_delivery_errors = false
+  # Randomize the order test cases are executed.
+  config.active_support.test_order = :random
```

Comment:
cool, didn't know you could do this, seems like a good idea for avoiding inadvertent test interdependencies.